Sources/Load.php

```diff
@@ -3846,17 +3846,32 @@ function set_avatar_data($data = array())
  */
 function get_auth_secret()
 {
-	global $auth_secret, $sourcedir, $smcFunc;
+	global $context, $auth_secret, $sourcedir, $smcFunc, $db_last_error, $txt;
```

Comment:
```suggestion
	global $context, $auth_secret, $sourcedir, $boarddir, $smcFunc, $db_last_error, $txt;
```

See below for explanation.

Reply:
Why do I need $boarddir here?  Its not being used here.